README.md

```diff
@@ -326,7 +326,7 @@ pub fn prove_work() -> (BaseElement, StarkProof) {
         0,  // grinding factor
         FieldExtension::None,
         8,   // FRI folding factor
-        128, // FRI max remainder length
+        127, // FRI max remainder length
```

Comment:
I think the comment is obsolete here as well. Should be "FRI remainder max degree".

Reply:
Correct, changed it in the following commit!